src/modules/shared/components/Menu/Menu.tsx

```diff
@@ -45,6 +46,11 @@ const _Menu = ({ tabs, className }: MenuProps) => {
   }/${year}`;
 
   const tabsDefault: TabsConfig[] = [
+    {
```

Comment:
Consegue colocar uma validação aqui por enquanto da node env para só aparecer se for development acho que essa daqui 
NEXT_PUBLIC_ENVIRONMENT